package.json

```diff
@@ -20,14 +15,11 @@
         }
     ],
     "scripts": {
-        "version": "npm run build-all",
+        "version": "npm run build",
```

Comment:
is this really needed?

Reply:
Yes this allows for automatically creating, committing, and tagging the new build with the updated package version numbers as part of running `lerna version`